the-collab-lab/tcl-16-smart-shopping-list
# Resolves 17. As a user, I want the modal alerts to match the theme styling of the rest of the app.

## Description

This code changes sweet alerts' [theme](https://sweetalert.js.org/docs/#theming) to match our app. I didn't change many of the colors because some of the alerts have icons and buttons have preset colors already, so I focused on the font and kept the colors for the button simple. Instead of adding the code to our base.scss file, I separated it to its own swal.scss file and instead imported it into base. 

I also changed all of the previous alerts to sweet alerts to remain consistent with the rest of our swal (specifically, for an invalid token/list and adding an item that already exists). Additionally, I made some UX driven updates to the swals that we already have; I increased the time that the pop-up for an item successfully added was present and other copy.

## Related Issue

closes #17 

## Acceptance Criteria

- should include colors and type treatment of wireframe

- should tweak timers and copy for best UX

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|  ✓ | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before

![Kapture 2021-02-09 at 19 15 55](https://user-images.githubusercontent.com/62153993/107445964-7e237f80-6b0b-11eb-9733-b98b7d3823c3.gif)

### After

[![Image from Gyazo](https://i.gyazo.com/b68778976fb3d0adcf74ca6278714c08.gif)](https://gyazo.com/b68778976fb3d0adcf74ca6278714c08)

[![Image from Gyazo](https://i.gyazo.com/e20fe780aafe6e74aef361946d58eac5.gif)](https://gyazo.com/e20fe780aafe6e74aef361946d58eac5)

## Testing Steps / QA Criteria

- switch to my branch ```wl-modal-styling```
- in your terminal ```npm start``` and open your browser to localhost:3000
- check the modal/alert when you:

1. put in an invalid token
2. add an item
3. add an item that already exists
4. delete an item
